.github/workflows/stale.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '39 7 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
```

Comment:
Is there a way to configure following?
```
        days-before-stale: 30
        stale-issue-label: stale
        stale-pr-label: stale
        exempt-issue-labels: lifecycle/frozen
        exempt-pr-labels:    lifecycle/frozen
        days-before-close: 30
        close-issue-message: This issue was automatically closed because of stale in 10 days
        close-pr-message: This PR was automatically closed because of stale in 10 days
```

Reply:
Thank you. I replicated the [config for stale-bot](https://github.com/kubeflow/pipelines/blob/master/.github/stale.yml). It's a little different from what you proposed in the comment above. Should we preserve the old behavior?